api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -467,6 +468,36 @@ message HttpConnectionManager {
   // with `prefix` match set to `/dir`. Defaults to `false`. Note that slash merging is not part of
   // `HTTP spec <https://tools.ietf.org/html/rfc3986>` and is provided for convenience.
   bool merge_slashes = 33;
+
+  // [#not-implemented-hide:]
+  // Configuration of local reply returned by Envoy. Allows to specify mappings and format of
+  // response.
+  LocalReplyConfig local_reply_config = 36;
+}
+
+message LocalReplyConfig {
+  // Configuration of list of mappers which allows to filter and change local response.
+  // The client will iterate through mappers until first match.
+  repeated ResponseMapper mapper = 1;
+
+  // Allows to define custom format of local reply.
+  // It is allowed to use :ref:`command operators <config_access_log_command_operators>`
+  // which will be resolved to actual data.
+  type.StringOrJson format = 2;
```

Comment:
Can the `format` be on per mapper basis, please?
Example: we want to return different body and/or content type depending on the generated status code.

Reply:
During config proposition we aggred that this might be added later. So I might add it in next PR.

Reply:
Have read the comments for prior version of pull request. It resolves the comment so no concerns from my side here.